### PR TITLE
Solution for the password issue

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ deploy:
...
+      secure: "Bzwk661qPFSBm14n3cOpaTR8ELei7Fw9lvKOGHlKyU/VpR3YB2qqF8eIsEmgzB4vpi6v6tpcZv/dpseSFn9NG4un75s5cZ1//yFp0KGaRGDcMM2W4+j3RTmHPrvp46X2z3lXGvwY+kfDTmv5px6+SuDpVDMKUgnk+wp7JvGG3FNItUK9WfF6MTwGCp4A/KIxLtKCCWxQhL3g8Gp8vNWBTXU6ZLrF/PtQyz8yZv3N01b86TfpM7DHYnZD4cCxam4YPyCzlOqE30kKYz9RkXSJCFepcBAni8kGkuGs5j34qf4IEck7E9/qWF2OOTtCa7mwxCnpIPl5uVhrMi+6XRMuRIr8P3+fuTGCrAhEeY1zdVX1wqrw0AgQNCfz288GLVZdbc4iV6MiZAPIwc4nNuP4WJBMCC1WUEsNK6a5GsVPNG1KP5Q2mVIQE1KJj9/Wv0LWglKnG2DA1hQd5Xu/hIUiMkzO0+8aYM0eM54AkHKCR9Drv90kTwcGhFYc7fQWVa4YXXVDzdfaOGn0EUR6ePN7Vg0KRKkDbHQ2yg4rZJsYg1tkRxd1XgGb1HCaeXZA7KVcwXIQXf4bkp/lRDholc81yYViCG5oR5wm1xdFIJVj7dP8Z55aPh081KbHVhUzXoTvEWnS9fOot0fG7DgPBDcVLpeMdmKLjid4no7lbyPB2qw="
```
Comment (niccokunzmann) [inline .travis.yml]: Is the submission of the key intended? Would it be better if we move this to the settings and just add a comment here that it should be set?

File: scripts/chroot.sh [hunk 2]
```diff
@@ -40,13 +40,6 @@ apt-get install -f
 apt-get -qq -y install xorg sddm
 apt-get -qq -y install lxqt
 
-#  may fix the black screen issue 
-apt-get -qq -y install xserver-xorg-video-intel
-
-# or alternative
-apt-get -qq -y install openbox pcmanfm-qt lxqt-admin lxqt-common lxqt-config lxqt-globalkeys lxqt-notificationd lxqt-panel lxqt-policykit lxqt-powermanagement lxqt-qtplugin lxqt-runner lxqt-session lxqt-sudo
```
Comment (niccokunzmann) [inline scripts/chroot.sh]: We are using plasma now, not LXQT, right?

File: scripts/chroot.sh [hunk 3]
```diff
@@ -139,19 +132,6 @@ apt-get -qq -y install oxygen-icon-theme
 # Install text editor
 apt-get -qq -y install kate
 
-# Remove lxqt-powermanagement
-#dependency on lxqt
-#apt-get -qq -y purge lxqt-powermanagement
-#apt-get -qq -y purge lxqt-powermanagement-l10n
```
Comment (niccokunzmann) [inline scripts/chroot.sh]: That can be removed, sure!

File: scripts/chroot.sh [hunk 4]
```diff
@@ -139,19 +132,6 @@ apt-get -qq -y install oxygen-icon-theme
...
-
-# temporary for debugging black screen issue
-cat /etc/default/grub
-cat /boot/grub/grub.cfg
-ls /boot/grub
-
-
-#Remove Kwin
-apt-get remove kwin
```
Comment (niccokunzmann) [inline scripts/chroot.sh]: This needs to be removed? Was that not something with the black screen?

File: scripts/chroot.sh [hunk 5]
```diff
@@ -172,14 +152,6 @@ dpkg -i --force-overwrite systemlock_0.1-1_all.deb
 apt-get install -f
 apt-get -qq -y remove dconf-tools
 
-# Instalation of packages from Generator Webapp
-SCRIPT_URL=https://www.github.com/fossasia/meilix-generator/archive/master.zip
```
Comment (niccokunzmann) [inline scripts/chroot.sh]: The removal of this is unrelated to the issue, I think.

File: scripts/chroot.sh [hunk 6]
```diff
@@ -190,9 +162,6 @@ rm meilix-metapackage_1.0-1_all.deb
 rm systemlock_0.1-1_all.deb plymouth-meilix-logo_1.0-1_all.deb plymouth-meilix-text_1.0-1_all.deb
 rm meilix-imclient_*_all.deb
 
-# Why was this added?
-# apt-get remove --purge wget apt-transport-https
```
Comment (niccokunzmann) [inline scripts/chroot.sh]: Can be removed!

File: scripts/chroot.sh (hunk 2 above)
Comment (tabesin) [reply scripts/chroot.sh]: sddm is the one for plasma and lxqt and it has known issues with password free login. I tried to install lubuntu-desktop along to get a login and see what is happening, That failed so far.

Idea, maybe we should create a default user in the chroot with addpwd. I have no idea how hotelos as a user gets generated.
